Annex60/Media/Air.mo

```diff
@@ -624,6 +717,7 @@ algorithm
   lambda := Modelica.Media.Incompressible.TableBased.Polynomials_Temp.evaluate(
       {(-4.8737307422969E-008), 7.67803133753502E-005, 0.0241814385504202},
    Modelica.SIunits.Conversions.to_degC(state.T));
+annotation(LateInline=true);
```

Comment:
The rational is that for using this quadratic polynomial, I don't expect the symbolic processor to do any manipulations. Hence, I suggest to leave the function call in the code until after the symbolic processing happened.